README.md

```diff
@@ -44,6 +44,12 @@ OPENEDX_API_BASE_URL=http://docker.for.mac.localhost:18000
 OPENEDX_BASE_REDIRECT_URL=http://edx.odl.local:18000
 ```
 
+### Create Homepage
+Go to `/cms/pages/` in your local deployment and create a `Home Page` type page at the root of the site.
+Then go to `CMS Admin > Settings > Sites > (default site)` and set this page as the "Root page" of the site.
+Go back to `/cms/pages/` and delete the default Wagtail home page titled "Welcome to your new Wagtail site!"
```

Comment:
Should not this include instructions about moving your existing pages under new home page ?

Reply:
Yes you can add a note.

Reply:
Can you also add a note that this should be done before running `./manage.py seed_data`?

Reply:
Comments added, although the migration in #395 moves everything under the home page anyway. There's also a management command in there as well that does the same thing if you need to manually invoke it later.